cookbooks/aws-parallelcluster-platform/resources/install_packages/install_packages_ubuntu22.rb

```diff
@@ -29,5 +29,5 @@ def default_packages
      r-base libblas-dev libffi-dev libxml2-dev
      libgcrypt20-dev libevent-dev iproute2 python3 python3-pip
      libatlas-base-dev libglvnd-dev iptables libcurl4-openssl-dev
-     coreutils moreutils curl python3-parted environment-modules)
+     coreutils moreutils curl python3-parted environment-modules libdbus-1-dev)
```

Comment:
question: should we do the same for other OSes in future?

Reply:
Another thing to consider is if we can move this libdbus-1-dev into the Slurm cookbook, since it's just required by Slurm.